src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb.model.ts

```diff
@@ -0,0 +1,14 @@
+export interface SMBCluster {
+  cluster_id: string;
+  auth_mode: string;
+  intent: string;
+  domain_settings?: DomainSettings;
+  user_group_settings?: string[];
+  custom_dns: string[];
+  placement: string;
```

Comment:
I believe there is this: `CephServicePlacement` that covers the placement custom type. We could use it here